CHANGELOG.rst

```diff
@@ -76,10 +100,7 @@ Version 0.8 (2020-09-06)
 
 **Acknowledgments**
 
-Thanks to everyone who contributed to this release!
-People with a "+" by their names contributed a patch for the first time.
-
-* Brendan Ward
+Thanks to everyone who Mike T
```

Comment:
something might have gone wrong here ;)

Reply:
Oops, thanks for looking back at this! Just fixed it in master.